docs/source/acquire.rst

```diff
@@ -0,0 +1,31 @@
+acquire
+-------
+
+Acquire can collect forensic artefacts from a live system or an image. Resulting images
+can be fed to other Dissect tools like target-query.
+
+To run **acquire** on a live system:
+
+.. code-block:: console
+    
+    $ acquire
+
+.. note::
+
+    To be able to access the full system to collect all its artefacts one must
+    run acquire with administrator privileges.
+
+
+Profiles
+~~~~~~~~
+
+By default, Acquire runs the full profile, meaning it will collect as much as it can.
+You can select specific artefacts you are interested in or you can select a profile.
+
+    
+.. note::
+
+    For a complete overview of all options see :doc:`here <tools/acquire>`.
```

Comment:
Why are there two acquire pages?

Reply:
This Acquire page is meant to give a simple overview. I expect this page to be extended with the core profiles, but not much else. The other page is a bit unwieldy and describes all the bells and whistles, which is okay, but I think a separation between the basic usage and an in-depth discussion would be nicer.